clash-ffi/example/run-iverilog.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# This is just a minimalistic script for demonstrating the process of
+# running the clash-ffi example using the Icarus Verilog VVP runtime
+# engine. The script is not designed to work in any possible system
+# environment and may not work immediatly for you. It is intended to
+# serve as an easy starter instead. Adapt it too you needs if it's not
+# working out-of-the-box for you.
+
+###############################
+
+# adapt these variables, if the tools are not in your PATH already
```

Comment:
I feel the transitive use of _adapt_ is only possible with a phrase starting with _to_. I'm not 100% sure, but I prefer _change_. Actually I don't like change, but I mean, in this sentence.

The comma can go.